src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/ShowCommand.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.logic.wanderlustlogic.wanderlustcommands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.wanderlustlogic.wanderlustcommands.exceptions.CommandException;
+import seedu.address.model.accommodation.Accommodation;
+import seedu.address.model.activity.Activity;
+import seedu.address.model.friend.Friend;
+import seedu.address.model.travelplanner.Model;
+
+/**
+ * Navigates and display the specified travel plan object based on user input.
+ */
+public class ShowCommand extends Command {
+    public static final String COMMAND_WORD = "show";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Show the respective travel plan object tab identified by the keyword.\n"
+            + "Parameters: travelPlanObjectType\n"
+            + "Example: " + COMMAND_WORD + " -activity";
+
+    public static final String MESSAGE_SHOW_SUCCESS = "show: %1$s";
+
+    private final String travelPlanObjectString;
+
+    private int travelPLanObjectType;
```

Comment:
travelPLanObjectType spelt with capital L by accident I think. Same throughout this file.

Reply:
ok thxtx